tmp.md

```diff
@@ -0,0 +1,1507 @@
+
```

Comment:
is this file meant to be formatted in some way that makes it easier to read?

Reply:
My comment was about the whole file and not a specific line, but I had to pick a link to make a comment.

Anyway, looks like it's a `|` separated file. Since it's called `tmp.md` I was thinking it's markdown. Maybe don't call it `*.md` then.

After opening it in excel and setting the custom delimiter to `|` it made sense.

Reply:
I was planning to use markdown, so I went with ‘|' as the separator.  Unfortuantely, neither the markdown plugin or the csv plugin that I tried for visual studio code worked well, so I just went with using excel.

In general I hate to use excel, however, I had already spent a day to produce this and had gotten the information that I needed from this file.  It is just here for documenting things and nothing else.

Reply:
> I was planning to use markdown, so I went with ‘|' as the separator. 

You need `----` to separate the headers in addition to '|' in a markdown table. https://www.markdownguide.org/extended-syntax/#tables

> It is just here for documenting things and nothing else.

If something is going to be added into a repository then it's purpose should be clear at least.

Reply:
ok, I'll delete this from the repo